src/core/metrics/app-metrics.ts

```diff
@@ -3,25 +3,52 @@ import { Sequence } from './sequence';
 class MetricMarker {
   readonly event: string;
   readonly timestamp: number;
-
   constructor(event: string) {
     this.event = event;
     this.timestamp = performance.now();
   }
 }
 
+class SessionSettings<F extends string> {
+  toggle(setting: F) {
+    window.sessionStorage.getItem(setting)
+      ? window.sessionStorage.removeItem(setting)
+      : window.sessionStorage.setItem(setting, 'true');
+  }
+
+  isEnabled(setting: F) {
+    return window.sessionStorage.getItem(setting) !== null;
+  }
+}
+
 export class AppMetrics {
   markers: MetricMarker[] = [];
+  maxLogSize = 100;
   private sequences: Set<Sequence> = new Set();
-  private eventLog: Set<string> = new Set();
+  private completed: Record<string, number> = {};
+  private eventLog: string[] = [];
+  private settings = new SessionSettings<'KONTUR_SQ_ALERT' | 'KONTUR_SQ_LOG'>();
 
   constructor() {
     globalThis.KONTUR_METRICS = {
       sequences: this.sequences,
       events: this.eventLog,
+      toggleAlert: () => this.settings.toggle('KONTUR_SQ_ALERT'),
+      toggleLog: () => this.settings.toggle('KONTUR_SQ_LOG'),
```

Comment:
Why theese needed for global window object?

Reply:
Mostly for testers and debugging. Testers and back-ends want to se is event work right on fe side, so I add such "debug tools"